src/Symfony/Bundle/SecurityBundle/Security.php

```diff
@@ -148,6 +155,17 @@ public function logout(bool $validateCsrfToken = true): ?Response
         return $logoutEvent->getResponse();
     }
 
+    /**
+     * Checks if the attribute is granted against the user and optionally supplied subject.
+     *
+     * This should be used over isGranted() when checking permissions against a user that is not currently logged in or while in a CLI context.
+     */
+    public function userIsGranted(UserInterface $user, mixed $attribute, mixed $subject = null): bool
```

Comment:
isUserGranted no ?

Reply:
👍 discussed internally, either this or `isGrantedForUser()`

Reply:
see #59214